src/Codeception/Coverage/SuiteSubscriber.php

```diff
@@ -61,6 +61,15 @@ protected function applySettings($settings)
                 $this->settings[$key] = $settings['coverage'][$key];
             }
         }
+
+        if (isset($settings['coverage']['advanced'])) {
+            foreach ($settings['coverage']['advanced'] as $key => $value) {
+                $method = 'set' . ucfirst($key);
+                if (method_exists($this->coverage, $method)) {
+                    $this->coverage->$method($value);
+                }
```

Comment:
I think that it would be good to throw exception about unknown option in `else` branch.

Reply:
Makes sense, will add it